modules/odc-grouping-multiple-components.adoc

```diff
@@ -11,20 +11,28 @@ The following procedure adds a MariaDB database service to an existing applicati
 
 .Prerequisites
 
-* Ensure that you have created and deployed a Node.js application on {product-title} using the *Developer* perspective.
+* You have created and deployed minimum two or more components on {product-title} using the *Developer* perspective.
```

Comment:
This entire module is being changed by Sri, please coordinate with her to reconcile it. Verify if this is causing the conflicts.

Reply:
deployed a minimum of two or more